src/pages/InboxPage/InboxDetailsPanel.jsx

```diff
@@ -33,6 +36,12 @@ const InboxDetailsPanel = ({ messages = [], selected = [], projectsInfo = {}, on
         entityType={entityType}
         entitySubTypes={entitySubType ? [entitySubType] : null}
         scope="inbox"
+        onWatchersUpdate={(added) => {
+          if (added.includes(user)) {
+            const name = selectedMessage.messages[0].origin.name
+            toast.success(`Future messages for ${name} will go to important.`)
```

Comment:
Slight change of wording.

```suggestion
            toast.success(`All future updates for ${name} will appear in your important inbox.`)
```